meson.build

```diff
@@ -0,0 +1,17 @@
+project(
+  'flatpak-xdg-utils', 'c',
+  version: '0.0.1',
+  license: 'LGPL-2.1+',
+)
+
+conf = configuration_data()
+conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
+conf.set_quoted('PACKAGE_VERSION', meson.project_version())
+config_h = configure_file(output: 'config.h', configuration: conf)
+
+add_global_arguments('-D_GNU_SOURCE=1', language: 'c')
```

Comment:
`add_project_arguments()`

Though you could argue this belongs in config.h anyway.

Reply:
Indeed, I forgot to move it to config.h when I added it for the other 2 options.

Thanks, fixed in the latest push to this branch.